discovery-infra/test_infra/helper_classes/cluster.py

```diff
@@ -545,3 +581,36 @@ def reset_cluster_and_wait_for_ready(self, cluster, nodes):
         cluster.wait_until_hosts_are_discovered()
         cluster.wait_for_ready_to_install()
 
+    def get_events(self, host_id=''):
+        return self.api_client.get_events(cluster_id=self.id, host_id=host_id)
+
+    def _find_event(self, event_to_find, reference_time, params_list, host_id):
+        events_list = self.get_events(host_id=host_id)
+        event_exist = False
+        for event in events_list:
+            if event_to_find in event['message']:
+                # Adding a 2 sec buffer to account for a small time diff between the machine and the time on staging
+                if utils.to_utc(event['event_time']) >= reference_time-2:
+                    if all(param in event['message'] for param in params_list):
+                        event_exist = True
+                        logging.info(f"Event to find: {event_to_find} exists with its params")
+                        return event_exist
+        return event_exist
```

Comment:
Use `for-else` instead of `event_exist` variable

```python
    def _find_event(self, event_to_find, reference_time, params_list, host_id):
        events_list = self.get_events(host_id=host_id)
        for event in events_list:
            if event_to_find in event['message']:
                # Adding a 2 sec buffer to account for a small time diff between the machine and the time on staging
                if utils.to_utc(event['event_time']) >= reference_time-2:
                    if all(param in event['message'] for param in params_list):
                        logging.info(f"Event to find: {event_to_find} exists with its params")
                        return True
        else:
            return False

Reply:
Done, please review